File: Beginner/The Smallest Pair/SmallestPair.java
/* package codechef; // don't place package name! */
import java.util.Scanner;
import java.util.Arrays;

/* Name of the class has to be "Main" only if the class is public. */
class SmallestPair {
	public static void main (String[] args) throws java.lang.Exception {
		// your code goes here
	    Scanner sc = new Scanner(System.in);
	    int n = sc.nextInt();
	    for (int i=0; i<n; i++) {
	        int arr_sz = sc.nextInt();
	        int arr[] = new int[arr_sz];
	        for (int j=0; j<arr_sz; j++) {
	            arr[j] = sc.nextInt();
	        }
            Arrays.sort(arr);
            System.out.println(arr[0]+arr[1]);
	    }
	}
}

File: Beginner/The Smallest Pair/SmallestPair.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int main() {
	// your code goes here
	int n;
	cin >> n;
	for (int i=0; i<n; i++) {
	    int arr_sz;
	    cin >> arr_sz;
	    int arr[arr_sz];
	    for (int j=0; j<arr_sz; j++) {
	        cin >> arr[j];
	    }
	    sort(arr, arr + arr_sz);
	    cout << arr[0] + arr[1] << endl;
	}
}
